node/cmd/guardiand/node.go

```diff
@@ -405,6 +411,7 @@ func init() {
 	suiMoveEventType = NodeCmd.Flags().String("suiMoveEventType", "", "Sui move event type for publish_message")
 
 	solanaRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "solanaRPC", "Solana RPC URL (required)", "http://solana-devnet:8899", []string{"http", "https"})
+	fogoRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "fogoRPC", "Fogo RPC URL (required)", "http://solana-devnet:8899", []string{"http", "https"})
```

Comment:
The default is the solana rpc?

Reply:
That's not a default, it's an example. https://github.com/wormhole-foundation/wormhole/blob/33dd6a56541c2d15d3e20faa1330ba542a5fa727/node/pkg/node/url_verification.go#L73